src/cli_config.rs

```diff
@@ -138,12 +148,15 @@ pub fn cli_parse() -> CliConfig {
 
     let motd = value_t!(matches.value_of("motd"), String).unwrap_or_else(|e| e.exit());
 
+    let lan_discovery_enabled = !matches.is_present("no-lan");
```

Comment:
why not `let lan_discovery_enabled = matches.is_present("lan");`?

Reply:
Because if neither of these args are specified default behavior - enable lan discovery.

Reply:
Maybe better use only one flag like `--no-lan`?

Reply:
Or `--lan=[enabled/disabled]` with default value = `enabled`

Reply:
I'd prefer just flag - it's simpler.

Reply:
I would prefer `--lan=[enabled/disabled]`, that gives more info about the default behaviour.